base-notebook/Dockerfile

```diff
@@ -92,6 +92,9 @@ RUN conda install --quiet --yes \
     fix-permissions $CONDA_DIR && \
     fix-permissions /home/$NB_USER
 
+# Initialize configuration file
+RUN jupyter notebook --generate-config
```

Comment:
I think we might need to do this as part of the prior command so that the `fix-permissions` script applies to the config file and parent directories in the user home. Otherwise, changes to the uid and gid will not allow proper access to the file.